src/components/button.tsx

```diff
@@ -7,7 +7,7 @@ interface Props {
   /** - */
   children: ReactNode
   dataTestid?: string
-  style?: "salmon" | "teal" | "teal-border"
+  style?: "salmon" | "teal" | "teal-border" | "white"
```

Comment:
What do you think about `white-border`? then we apply the same understanding that we have for the `teal-border`